pkg/git/git_test.go

```diff
@@ -8,14 +8,14 @@ func TestRemoteURL(t *testing.T) {
 	t.Parallel()
 
 	actual := RemoteURL("example.com", "app")
-	expected := "ssh://git@example.com:2222/app.git"
+	expected := "ssh://git@example-builder.com:2222/app.git"
```

Comment:
@bacongobbler, I think I mindlessly updates that test case without realizing it was at least an _edge_ case, or possibly even an _invalid_ case.

In the typical case, the controller would be at the `deis.example.com`, which, according to this logic, would make `deis-builder.example.com` the subdomain for the builder.

I'm trying to think of a scenario where the controller _isn't_ exposed as some subdomain of the platform domain.  From the router's perspective, the controller is "just another app," which is named `deis`, so it automatically occupies the `deis` subdomain of the platform domain (e.g. `example.com`).  Like other apps, it _could_ have a custom / vanity URL such as `mydeis.com`, but that's only achieved through some fairly major modifications to the charts...

I'm debating whether this edge case is worth addressing vs whether is is _so "edgy"_ that I should just update this test case to be more realistic.

Thoughts?